src/main/resources/application.properties

```diff
@@ -2,3 +2,9 @@ server.port=8135
 
 # Custom properties
 com.ampnet.web3provider.provider.blockchain-api=https://eth-mainnet.alchemyapi.io/v2/hzYkWsp9ig3umpP6-ol3h2F64nof0lQ8
+
+# Redis properties
+com.ampnet.web3provider.provider.redis.eth-get-balance-ttl=5
+com.ampnet.web3provider.provider.redis.eth-get-code-ttl=5
+com.ampnet.web3provider.provider.redis.eth-chain-id-ttl=30 * 86400
```

Comment:
Does this actually work like this? I have never seen math operations in properties file before.

Reply:
I also didn't know but it works, I have checked it.